config.ru

```diff
@@ -1,6 +1,6 @@
 require "rack-timeout"
 use Rack::Timeout           # Call as early as possible so rack-timeout runs before other middleware.
-Rack::Timeout.timeout = 20
+Rack::Timeout.timeout = 6
```

Comment:
actually i take that back - you might want to double check whether or not rack timeout includes time that the request spent queued up before actually being serviced.  if it doesn't, then 6 might make more sense (to pad for that potential latency).